### ISSUE TITLE
Add nuke damage to infoboxes

### ISSUE THREAD

**Issue by CornHusker89** [Galaxypedia-Wiki/ketchupbot-updater#38]
It was suggested that we use the nuke size to calculate nuke damage at various distances, and put those values in applicable infoboxes. While it's not necessary and users can calculate it themselves, I think it could be a helpful addition

**Comment by CubdWolf** [comment]
I was the one who suggested this, and I think that (realistically) there are only 2 ways this idea could go:

1: It could be dropped, just add the formula to the nukes page, and leave it at that (My tought process is that during the nuke of something considerable (Ie: SC or base), players will be focused on warping out of there ASAP, not looking up wether or not they can actually tank the nuke using the formula that is provided (Even with a ridiculousy easy UI, most wouldn't even consider it)).

2: Have the system be EXTREMELY user friendly, this could be useful for people just being curious, and not necessarily used in the heat of the situation. (Being honest, most people wouldn't even plug in values to calculate the damage they would get from a nuke, let alone use a formula to calculate it by hand).

TL:DR : Adding this would probably only serve curious people. It could be fun to add if it isin't too hard to implement, but the minute that it takes considerable time/effort, I think that it should be dropped, as it wouldn't be worth investing into (More impotant issues are already on hold).

**Comment by CornHusker89** [comment]
It's not crazy hard to add, but it was decided that changing galaxy info to return nuke damage at set distances would be a better than ketch calculating it. Then the discussion became, "But what distances?", to which there was no clear consensus.

As far as a UI element for it in the infobox (which is what I'm assuming you want) like a slider or something, that would take more effort than it's worth. If this were to happen, it would likely be just a few set distances for which nuke damage is calculated, and displayed in the infobox under the explosion size.

**Comment by CubdWolf** [comment]
"As far as a UI element for it in the infobox (which is what I'm assuming you want) like a slider or something, that would take more effort than it's worth." That is what I meant. So I assume that this idea can be dropped, since, as you said, it is not worth the effort to imprement it.

**Comment by smallketchup82** [comment]
i'll close this as not planned for now. we can reopen/revisit later
